jena-fuseki2/jena-fuseki-webapp/src/test/java/org/apache/jena/fuseki/TestMetrics.java

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Add TestMetrics.class to \@Suite.SuiteClasses in TS_FusekiWebapp.

Reply:
Changed :+1: